wpiunits/src/main/java/edu/wpi/first/units/Voltage.java

```diff
@@ -4,6 +4,15 @@
 
 package edu.wpi.first.units;
 
+/**
+ * Unit of voltage dimension.
```

Comment:
"Unit of electric voltage dimension", maybe, to match the suggested documentation for `Current.java`